lib/services/rating_service.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:project_eureka_flutter/models/rating_model.dart';
+import 'package:flutter_dotenv/flutter_dotenv.dart' as DotEnv;
+import 'dart:async';
+import 'dart:convert';
+import 'package:http/http.dart' as http;
+
+class RatingService {
+  Future<RatingModel> updateRating(RatingModel rating) async {
+    await DotEnv.load();
+
+    final response = await http.put(
+      Uri.http(
+        DotEnv.env['HOST'] + ':' + DotEnv.env['PORT'],
+        '/v1/users/${rating.id}/rating',
+      ),
+      headers: <String, String>{
+        'Content-type': 'application/json; charset=UTF-8',
+      },
+      body: jsonEncode(
+        rating.toJson(),
+      ),
+    );
+
+    if (response.statusCode == 200) {
+      final body = json.decode(response.body);
+      RatingModel rating = RatingModel.fromJSON(body['ratings']);
+      return rating;
+    } else {
+      throw Exception('Failed to load ratings.');
+    }
```

Comment:
You need to use UserModel here instead of RatingModel as I mentioned before. However it is not necessary because it doesn't return anything (after you change it to void function). You could simply make a print statement in console that user was rated successfully and server responded with status 200.